org/branchprotection.yml

```diff
@@ -79,3 +79,21 @@ branch-protection:
             required_approving_review_count: 1
             bypass_pull_request_allowances:
               teams: ["wg-app-runtime-platform-bots"]
+
+        # CF on K8s repos to skip branch protection on
+        cloudfoundry/cf-k8s-secrets:
+          protect: false
+        cloudfoundry/eirini:
```

Comment:
eirini* and yttk8smatchers repos can be removed here as they get archived by #636 and removed from the WG charter yaml.

Reply:
Done!